cpu/stm32/Makefile.include

```diff
@@ -90,6 +90,16 @@ ifeq (,$(filter STM32MP157Cxx STM32F030x4,$(CPU_LINE)))
   BUILDDEPS += $(STM32IRQS_INCLUDE_FILE)
 endif
 
+F2_WITH_BKPRAM := STM32F205xx STM32F207xx STM32F215xx STM32F217xx
+F4_WITH_BKPRAM := STM32F405xx STM32F407xx STM32F415xx STM32F417xx \
+                  STM32F427xx STM32F429xx STM32F437xx STM32F439xx \
+                  STM32F446xx STM32F469xx STM32F479xx
+F7_WITH_BKPRAM := STM32F7xxxx
+ifneq (,$filter $(F2_WITH_BKPRAM) $(F4_WITH_BKPRAM) $(F7_WITH_BKPRAM),$(CPU_LINE))
```

Comment:
Is there some way to only maintain this list once?
Also why is `CPU_LINE` checked here and `CPU_MODEL` in `Makefile.features`?

Reply:
If you think it is ok to define `BACKUP_RAM_ADDR` and `BACKUP_RAM_LEN` in `Makefile.features`, then we could only maintain that list. But I thought it would not be appropriate because they are no features.